### PR TITLE
Reduce unnecessary logging in tests

### PR DESCRIPTION
**Pull request by martint**
## Release notes

(x) This is not user-visible or is docs only, and no release notes are required.

### CODE REVIEW

**Review by dain — APPROVED** [review]

File: lib/trino-hive-formats/pom.xml [hunk 1]
```diff
@@ -100,6 +100,12 @@
             <artifactId>modernizer-maven-annotations</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>io.airlift</groupId>
+            <artifactId>log-manager</artifactId>
+            <scope>provided</scope>
```
Comment (electrum) [inline lib/trino-hive-formats/pom.xml:106]: I think `runtime` is the correct scope for dependencies that are used in test code but are also needed transitively at runtime. `provided` scope makes it available on the compile classpath, but these shouldn't be on the compile classpath since they aren't referenced in production code.

File: plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/AccumuloQueryRunner.java [hunk 2]
```diff
@@ -29,6 +30,7 @@
 
 import java.util.Map;
 
+import static io.airlift.log.Level.OFF;
```
Comment (electrum) [inline plugin/trino-accumulo/src/test/java/io/trino/plugin/accumulo/AccumuloQueryRunner.java:33]: Should we be consistent about static importing `Level` values? I'm fine either way, but would prefer consistency.

File: plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraQueryRunner.java [hunk 3]
```diff
@@ -24,13 +25,20 @@
...
 import static io.trino.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
 import static io.trino.testing.QueryAssertions.copyTpchTables;
 import static io.trino.testing.TestingSession.testSessionBuilder;
 
 public final class CassandraQueryRunner
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("com.datastax.oss.driver.internal", OFF);
+        logging.setLevel("io.trino.plugin.cassandra.CassandraServer", OFF);
```
Comment (electrum) [inline plugin/trino-cassandra/src/test/java/io/trino/plugin/cassandra/CassandraQueryRunner.java:39]: This is our own code, so we can make them `log.debug()` instead of disabling logging.

File: core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java [hunk 4]
```diff
@@ -139,6 +141,11 @@
 public class TestingTrinoServer
         implements Closeable
 {
+    static {
+        Logging logging = Logging.initialize();
+        logging.setLevel("io.trino.event.QueryMonitor", Level.OFF);
```
Comment (electrum) [inline core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java:146]: We should make all these WARN or ERROR unless we need absolutely need them to be OFF.

File: plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixPlugin.java [hunk 5]
```diff
@@ -28,7 +28,12 @@ public void testCreateConnector()
     {
         Plugin plugin = new PhoenixPlugin();
         ConnectorFactory factory = getOnlyElement(plugin.getConnectorFactories());
-        factory.create("test", ImmutableMap.of("phoenix.connection-url", "jdbc:phoenix:test"), new TestingConnectorContext())
+        factory.create(
+                "test",
+                        ImmutableMap.of(
```
Comment (electrum) [inline plugin/trino-phoenix5/src/test/java/io/trino/plugin/phoenix5/TestPhoenixPlugin.java:33]: Formatting is off

**Review by electrum — APPROVED** [review]

File: lib/trino-hive-formats/pom.xml (hunk 1 above)
Comment (martint) [reply lib/trino-hive-formats/pom.xml:106]: I think I tried that but it didn't work. Let me try again just in case I missed it.

File: core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java (hunk 4 above)
Comment (martint) [reply core/trino-main/src/main/java/io/trino/server/testing/TestingTrinoServer.java:146]: I'll see what happens with WARN or ERROR, but in general, the problem is that many parts of the code generate logs at those levels for recoverable events, and in situations encountered normally (e.g., server shutdown, etc).
